tests/functional/spec/features/calculated_summary.spec.js

```diff
@@ -104,25 +105,22 @@ describe("Feature: Calculated Summary", () => {
       expect($(CurrencyTotalPlaybackPageWithFourth.calculatedSummaryAnswer()).getText()).to.contain("£40.71");
     });
 
-    it("Given I leave an answer empty, When i get to the currency summary, Then I should see no answer provided and new total", () => {
-      $(CurrencyTotalPlaybackPageWithFourth.fourthNumberAnswerEdit()).click();
-      $(FourthNumberBlockPage.fourthNumber()).setValue("");
-      $(FourthNumberBlockPage.submit()).click();
+    it("Given I leave an answer empty, When I get to the currency summary, Then I should see no answer provided and new total", () => {
```

Comment:
Not sure this one is clear. Looks like you are editing a question and removing its value, and because of that the total changes. The description reads as if this was the first pass through the questionnaire

Reply:
Haven't really change the existing behaviour, just removed the fourth and a half block being edited so that we have 2 different total when skipping / no skipping 4th questions. It may need a general refactor regardless of my changes.

Reply:
As it's me and although you have touched it, I will let you off